src/main/java/org/jboss/msc/service/ServiceContainerImpl.java

```diff
@@ -844,7 +844,7 @@ public void uncaughtException(final Thread t, final Throwable e) {
             ServiceLogger.ROOT.uncaughtException(e, t);
         }
     };
-    private static final ThreadPoolExecutor.CallerRunsPolicy POLICY = new ThreadPoolExecutor.CallerRunsPolicy();
+    private static final ThreadPoolExecutor.AbortPolicy POLICY = new ThreadPoolExecutor.AbortPolicy();
```

Comment:
We never want this policy unless we can go through and prove that we have 100% correct behavior when ```execute()``` throws an exception.